src/ui/common/src/components/workflows/WorkflowSettings.tsx

```diff
@@ -226,6 +232,28 @@ type WorkflowSettingsProps = {
   onClose: () => void;
 };
 
+// Returns whether `updated` is different from `existing`.
+function IsNotificationSettingsMapUpdated(
```

Comment:
I wish Typescript had a built in function to do this, but I'm not aware that it does :)